arm/Microsoft.Compute/virtualMachineScaleSets/deploy.bicep

```diff
@@ -322,7 +322,7 @@ param managedIdentityType string = ''
 @description('Optional. The list of user identities associated with the virtual machine scale set. The user identity dictionary key references will be ARM resource ids in the form: \'/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}\'.')
 param managedIdentityIdentities object = {}
 
-var publicKeysFormatted = [for item in publicKeys : {
+var publicKeysFormatted = [for item in publicKeys: {
```

Comment:
Replace `item` with `publicKey`
e.g. `for publicKey in publicKeys: `

Reply:
Fixed. Please check and resolve if you agree.